packages/ocean-lending/src/clients/oToken.ts

```diff
@@ -153,6 +161,27 @@ class OToken extends Client {
   public async interestRateModel(): Promise<string> {
     return this.contract.methods.interestRateModel().call();
   }
+
+  public async borrowInterest(address: string): Promise<number> {
+    const [exchangeRateCurrent, borrowAmount] = await Promise.all([this.exchangeRate(), this.borrowBalanceCurrent(address)]);
+
+    const mantissa = this.COMPOUND_BASE_DECIMALS + this.UNDERLYING_DECIMALS - this.OTOKEN_DECIMALS;
+    const oneOTokenInUnderlying = exchangeRateCurrent / Math.pow(10, mantissa);
+    const underlyingTokensAfter = Number(borrowAmount) * oneOTokenInUnderlying;
```

Comment:
why not keep as big int? Maybe we should keep the interest as big int and let users of client to convert the types